DQM/Integration/python/clients/pixel_dqm_sourceclient-live_cfg.py

```diff
@@ -73,6 +73,7 @@
 # Real data raw to digi
 process.load("EventFilter.SiPixelRawToDigi.SiPixelRawToDigi_cfi")
 process.siPixelDigis.IncludeErrors = True
+process.siPixelDigis.UsePhase1 = True
```

Comment:
sorry @davidlange6  I did not see this comment before, my bad. 
The era does not work out of the box, I get the following error:

python encountered the error: <type 'exceptions.AttributeError'>
'Process' object has no attribute 'gtEvmDigis'

I'll try to fix it in the coming days